### PR TITLE
Ensure gogoduck always gets spatial subset specified with at least on…

### PR DESCRIPTION
**Pull request by jonescc**
…e decimal place

### CODE REVIEW

File: src/test/javascript/portal/cart/GogoduckV1DownloadHandlerSpec.js [hunk 1]
```diff
@@ -61,7 +61,7 @@ describe('Portal.cart.GogoduckV1DownloadHandler', function () {
                         isNcwmsParams: true,
                         dateRangeStart: moment.utc('2000-01-01T01:01:01'),
                         dateRangeEnd: moment.utc('2014-12-23T23:59:59'),
-                        latitudeRangeStart: -42,
+                        latitudeRangeStart: -42.35,
```
Comment (danfruehauf) [inline src/test/javascript/portal/cart/GogoduckV1DownloadHandlerSpec.js]: Was the test wrong in that case?

Comment (danfruehauf) [inline src/test/javascript/portal/cart/GogoduckV1DownloadHandlerSpec.js]: Ah, never mind, that's the input. Forget about it.

File: web-app/js/portal/prototypes/Number.js [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Number.prototype.toDecimalString = function() {
+	return this.toString().contains('.') ? this.toString() : this + '.0';
```
Comment (danfruehauf) [inline web-app/js/portal/prototypes/Number.js]: So just what I did in GGD :)

File: src/test/javascript/portal/cart/GogoduckV1DownloadHandlerSpec.js (hunk 1 above)
Comment (jonescc) [inline src/test/javascript/portal/cart/GogoduckV1DownloadHandlerSpec.js]: Thought I should have something with a decimal place

**Pull request merged**